### PR TITLE
Update zh-Hans.hjson

### PR DESCRIPTION
**Pull request by GamMa0892**
Chinese Localization update

### CODE REVIEW

File: Localization/zh-Hans.hjson [hunk 1]
```diff
@@ -2002,15 +2002,15 @@ Mods: {
 			}
 
 			ShyUnderlingItem: {
-				// DisplayName: Dented Coin
-				// Tooltip: Summons a shy underling to fight for you
-				// Flavor: "'This coin has been used as target practice for a long time'"
+				DisplayName: 凹陷硬币
+				Tooltip: 召唤害羞的下属为你而战
+				Flavor: "这枚硬币长久以来一直被用做射击练习。"
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:2007]: original text is `"'This coin has been used as target practice for a long time'"` which ingame retains the quotation marks like this: `'This coin has been used as target practice for a long time'`, this means the translation needs to include them aswell, likely those from chinese typefont: `"“text here”"`. The added `"` in english are used whenever there are special characters at the start, such as `'`

File: Localization/zh-Hans.hjson [hunk 2]
```diff
@@ -2002,15 +2002,15 @@ Mods: {
...
 			}
 
 			SeriousUnderlingItem: {
-				// DisplayName: Well-Loved Doll
-				// Tooltip: Summons a serious underling to fight for you
-				// Flavor: "'A simple voodoo doll made of loose straw and hay.'"
+				DisplayName: 人气娃娃
+				Tooltip: 召唤一名认真的下属为你而战
+				Flavor: "由松散的稻草和干草做成的简单巫毒娃娃。"
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:2013]: Same as above with the `"“text here”"`

File: Localization/zh-Hans.hjson [hunk 3]
```diff
@@ -2080,7 +2080,7 @@ Mods: {
 				ReducedDamageTakenBoss: "{1}造成的伤害减少{0}%"
 				BuffImmunity: 当{1}存活时免疫{0}
 				TimesDied: 死亡次数: {0}/{1}
-				// DeleteFromInventory: You won't be needing the {0} anymore...
+				// DeleteFromInventory: 你不再需要{0}了……
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:2083]: Forgot to remove `// `

File: Localization/zh-Hans.hjson [hunk 4]
```diff
@@ -3094,383 +3094,383 @@ Mods: {
...
-						// 0Serious: Hey.
-						// 1Shy: ...yes?
-						// 2Serious: Would you like a doll?
-						// 3Shy: A doll?
-						// 4Serious: Yes. To calm your nerves.
-						// 5Shy: ...yes, please.
-						// 6Serious: I'll prepare it later then.
+						0Serious: 嘿。
+						1Shy: …是？
+						2Serious: 想要来个娃娃吗?
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:3438]: `? -> ？`

File: Localization/zh-Hans.hjson [hunk 5]
```diff
@@ -3094,383 +3094,383 @@ Mods: {
...
-						// 1Serious: Yes?
-						// 2Shy: Y-your dolls...
-						// 3Serious: ?
-						// 4Shy: They're cute...
-						// 5Serious: Thank you.
+						0Shy: 那个…不好意思……
+						1Serious: 怎么了？
+						2Shy: 那、那个…你的娃娃……
+						3Serious: ？
+						4Shy: They're cute...
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:3463]: Forgot

File: Localization/zh-Hans.hjson [hunk 6]
```diff
@@ -3483,8 +3483,8 @@ Mods: {
 		}
 
 		UI: {
-			// CircleUIToUnlock: To unlock: {0}
-			// HoverNPCUICatchWithNet: Catch it with a net
+			CircleUIToUnlock: 解锁: {0}
```
Comment (direwolf420) [inline Localization/zh-Hans.hjson:3486]: instead of `: `, use `：`

**Review by direwolf420 — CHANGES_REQUESTED** [review]
Thanks, great work! Here's some suggested improvements

**Review by GamMa0892 — COMMENTED** [review]
done. ⎝(◕u◕)⎠

**Pull request merged**